src/hmem_rocr.c

```diff
@@ -122,6 +122,10 @@ struct hsa_ops {
 	hsa_status_t (*hsa_iterate_agents)(
 		hsa_status_t (*callback)(hsa_agent_t agent, void *data),
 		void *data);
+	hsa_status_t (*hsa_system_get_info)(hsa_system_info_t attribute, 
+					    void* value);
+	hsa_status_t (*hsa_amd_portable_export_dmabuf)(const void* ptr, size_t size, 
```

Comment:
the hmem component has two modes of operations: one is where the rocr library has been linked to libfabric, the second one is when the libhsa-runtime64 is opened dynamically using dlopen(). In the second case, you need to maintain pointers to the individual functions that you want to access. I.e. you cannot call for example hsa_init() directly because the libfabric library has not been linked against libhsa-runtime64, but once you found the function hsa_init in the library that you opened with dlopen() (see https://github.com/ofiwg/libfabric/blob/main/src/hmem_rocr.c#L731), you can still call hsa_init but you need to remember where its located in the library. That is what the hsa_ops structure does